wpilibcExamples/src/main/cpp/examples/DifferentialDriveBot/include/DifferentialDrive.h

```diff
@@ -0,0 +1,76 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#pragma once
+
+#include <units/units.h>
+
+#include <frc/AnalogGyro.h>
+#include <frc/Encoder.h>
+#include <frc/Spark.h>
+#include <frc/SpeedControllerGroup.h>
+#include <frc/controller/PIDController.h>
+#include <frc/kinematics/DifferentialDriveKinematics.h>
+#include <frc/kinematics/DifferentialDriveOdometry.h>
+#include <wpi/math>
+
+class DifferentialDrive {
```

Comment:
This could be confusing because we have a class called DifferentialDrive already in the library.  I think this should be renamed and maybe we should use that class unless it does not make sense.

Reply:
We could rename the subsystem to "Drivetrain".

Reply:
Rename all the subsystems to `Drivetrain` sounds good. The name of the example will clearly convey what drivetrain type we're talking about.